dashboard/src/app/(landing)/components/pricingSection.tsx

```diff
@@ -0,0 +1,195 @@
+'use client';
+
+import { useCallback, useState } from 'react';
+import { Check } from 'lucide-react';
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from '@/components/ui/card';
+import { Badge } from '@/components/ui/badge';
+import { Button } from '@/components/ui/button';
+import Link from 'next/link';
+
+const eventRanges = [
+  { value: 10000, label: '10K', price: 0 },
+  { value: 50000, label: '50K', price: 6 },
+  { value: 100000, label: '100K', price: 13 },
+  { value: 150000, label: '150K', price: 20 },
+  { value: 200000, label: '200K', price: 27 },
+  { value: 500000, label: '500K', price: 48 },
+  { value: 1000000, label: '1M', price: 69 },
+  { value: 2000000, label: '2M', price: 132 },
+  { value: 5000000, label: '5M', price: 209 },
+  { value: 10000000, label: '10M', price: 349 },
+  { value: 25000000, label: '10M+', price: 'Custom' },
+];
```

Comment:
Quibble: For readability I would use `_`
```suggestion
const eventRanges = [
  { value: 10_000, label: '10K', price: 0 },
  { value: 50_000, label: '50K', price: 6 },
  { value: 100_000, label: '100K', price: 13 },
  { value: 150_000, label: '150K', price: 20 },
  { value: 200_000, label: '200K', price: 27 },
  { value: 500_000, label: '500K', price: 48 },
  { value: 1_000_000, label: '1M', price: 69 },
  { value: 2_000_000, label: '2M', price: 132 },
  { value: 5_000_000, label: '5M', price: 209 },
  { value: 10_000_000, label: '10M', price: 349 },
  { value: 25_000_000, label: '10M+', price: 'Custom' },
];
```